rulesets/data/es_lookup_by_cz.csv

```diff
@@ -0,0 +1,276 @@
+type,subtype,program,1A,1B,1C,2A,2B,2C,3A,3B,3C,4A,4B,4C,5A,5B,5C,6A,6B,6C,7,8
+radiant_barrier_bool,has_radiant_barrier,SF_National_3.0,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,SF_National_3.0,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,SF_National_3.1,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,SF_National_3.2,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,has_radiant_barrier,SF_Pacific_3.0,TRUE,TRUE,TRUE,,,,,,,,,,,,,,,,,
+radiant_barrier_bool,,SF_Pacific_3.0,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,
+radiant_barrier_bool,,SF_Florida_3.1,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,,,,,,,,,,,,,,
+radiant_barrier_bool,,SF_OregonWashington_3.2,,,,,,,,,,,,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,,
+radiant_barrier_bool,has_radiant_barrier,MF_National_1.0,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,MF_National_1.0,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,MF_National_1.1,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,MF_National_1.2,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE
+radiant_barrier_bool,,MF_OregonWashington_1.2,,,,,,,,,,,,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,FALSE,,
```

Comment:
This is pretty complicated. Let's make it a two step process: 1) in the ruleset, determine whether the appropriate condition is met (e.g., vented attic exists), and then 2) look up true/false in this file for the climate zone-specific decision.

More generally, I don't think we need 100% of the logic to be encapsulated in the lookup files. Some complicated logic may still have to remain in the ruleset.